products/debian12/profiles/anssi_bp28_enhanced.profile

```diff
@@ -25,11 +25,12 @@ selections:
   - '!selinux_state'
   - '!audit_rules_mac_modification'
   - '!selinux_policytype'
-  - 'apparmor_configured'
-  - 'all_apparmor_profiles_enforced' 
-  - 'grub2_enable_apparmor'
-  - 'package_apparmor_installed'
-  - 'package_pam_apparmor_installed'
+  - '!sebool_selinuxuser_execheap'
```

Comment:
To fix the CI you will need to do the same to RHEL8-RHEL10 profiles.

Reply:
I will update RHEL profiles in a few days. Thank you for your review.